tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1080,6 +1080,11 @@ platform_tests/sfp/test_sfputil.py::test_check_sfputil_reset:
       - "(platform in ['x86_64-cel_e1031-r0'] or 't2' in topo_name) and https://github.com/sonic-net/sonic-mgmt/issues/6218"
       - "asic_type in ['nvidia-bluefield']"
 
+platform_tests/test_advanced_reboot.py:
+  skip:
+    reason: "Skip advanced reboot on vms20-t0-7050cx3-2 since no SSD on this device and failures to session teardown"
+    conditions: "topo_name in 'vms20-t0-7050cx3-2'"
```

Comment:
`topo_name` and testbed name are two different things. I don't think this should work and I don't know why it worked as shown in description.

Reply:
Ah, I think it worked as it will skip the test on a lot of testbeds, other than this one:

```
>>> 
>>> topo_name='t0'  
>>> topo_name in 'vms20-t0-7050cx3-2'
True
>>> 
```

Reply:
I changed to skip the test in `advanced_reboot` module to check if the testbed name is exactly `vms20-t0-7050cx3-2`, ran twice on different 7050 devices and attached the results above, I think it works in this way.